charts/steward/values.yaml

```
@@ -0,0 +1,78 @@
+# Default values for steward.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+targetNamespace:
+  create: true
+  name: "steward-system"
+
+runController:
+  image:
+    repository: stewardci/stewardci-run-controller
+    tag: 200114_1b68057
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 1
+      memory: 64Mi
+    requests:
+      cpu: 50m
+  podSecurityContext: {}
+    # fsGroup: 2000
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  nodeSelector: {}
+  affinity: {}
+  tolerations: []
+
+tenantController:
+  image:
+    repository: stewardci/stewardci-tenant-controller
+    tag: 200114_1b68057
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 1
+      memory: 32Mi
+    requests:
+      cpu: 10m
+  podSecurityContext: {}
+    # fsGroup: 2000
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  nodeSelector: {}
+  affinity: {}
+  tolerations: []
+
+# imagePullSecrets are used to pull controller images, but no other images.
+imagePullSecrets: []
+
+# pipelineRuns contains common configuration for all pipeline runs
+pipelineRuns:
+  logging:
+    elasticsearch:
+      # indexURL is the URL of the Elasticsearch index to send logs to.
+      # If null or empty, logging to Elasticsearch is disabled.
+      # Example: http://elasticsearch-master.elasticsearch.svc.cluster.local:9200/jenkins-logs/_doc
+      indexURL: ""
+  jenkinsfileRunner:
+    image:
+      repository: stewardci/stewardci-jenkinsfile-runner
+      tag: 200108_c7477b7
```

Latest version (at the moment)
```suggestion
      tag: 200117_9442211
```

---

This will be update with the next merge of master.